taxi/views.py

```diff
@@ -79,6 +86,19 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+class CarDriverManageView(LoginRequiredMixin, generic.View):
```

Comment:
Using a static method for handling POST requests in `CarDriverManageView` is unconventional. Consider using instance methods to handle requests, which allows better integration with Django's request handling and middleware.